SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialGameInstance.h

```diff
@@ -37,7 +37,10 @@ class SPATIALGDK_API USpatialGameInstance : public UGameInstance
 private:
 
 	// TODO: Move SpatialConnection ownership to NetDriver
+	// TODO remove this
```

Comment:
This is still needed right?

Reply:
Eh we can just have an accessor. If other people want to use the connection that's fine.